### PR TITLE
[FIX]: Handling the minimum of users error

### LINKED ISSUE #254: [FIX]: Handling the minimum of users error
## Description

When you make the first login/register in the app and try create a new board, you need to have a minimum of 4 users.

image

### Steps to reproduce

1. Register/Login in the app;
2. Click on **Create board** button (on Dashboard or Boards page);

### Expected result

Show the error alert as appears to the team.

[Design](https://www.figma.com/file/McDBByDqFzjYuYZ4CwVpax/UI-v2.0?node-id=1485%3A129850)

- [x] Alert already created 

### Actual result

Appears an error from Next.js

## Idea to solution

Use the `createBoardError` atom to catch the error and set to true, this way, it's appear a alert in the create board page.

> **Note:** alert already created on `[boardId].ts` page, only need to set this state for this particularly error.

### Use atom Value
```ts
const haveError = useRecoilValue(createBoardError);
```

### Use atom state
```ts
const [haveError, setHaveError] = useRecoilState(createBoardError);
```
> I use this to set the error on `TeamSubTemasConfigurations.tsx` component.

### PR DESCRIPTION
**Pull request by joaosantos99**
Fixes #254

## Screenshots (if visual changes)

Screenshot 2022-06-24 at 16 49 46

## Proposed Changes

  - Added the error alert when when the team have less than 4 members
  - Stopped the creation of multiple boards ( cause of the issue error ) when the team have less than 4 members

This pull request closes #254

### CODE REVIEW

**Review by nunocaseiro — APPROVED** [review]

File: frontend/src/components/CreateBoard/SubTeamsTab/TeamSubTeamsConfigurations.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ const TeamSubTeamsConfigurations: React.FC = () => {
 	const setBoardData = useSetRecoilState<CreateBoardData>(createBoardDataState);
 	const setToastState = useSetRecoilState(toastState);
 	const [haveError, setHaveError] = useRecoilState(createBoardError);
+	const minMembers = 4;
```
Comment (dsousa12) [inline frontend/src/components/CreateBoard/SubTeamsTab/TeamSubTeamsConfigurations.tsx:33]: Not a problem, but as `minMembers` is a constant value maybe declare using uppercase letters (`MIN_MEMBERS`)

File: frontend/src/hooks/useCreateBoard.tsx [hunk 2]
```diff
@@ -15,6 +15,7 @@ const useCreateBoard = (team: Team, stakeHolders: string[]) => {
 	const { board } = createBoardData;
 
 	const minTeams = 2;
+	const minMembers = 4;
```
Comment (dsousa12) [inline frontend/src/hooks/useCreateBoard.tsx:18]: Same:
> Not a problem, but as minMembers is a constant value maybe declare using uppercase letters (MIN_MEMBERS)

**Review by dsousa12 — CHANGES_REQUESTED** [review]

File: frontend/src/components/CreateBoard/SubTeamsTab/TeamSubTeamsConfigurations.tsx (hunk 1 above)
Comment (joaosantos99) [reply frontend/src/components/CreateBoard/SubTeamsTab/TeamSubTeamsConfigurations.tsx:33]: It makes sense I will proceed to change

**Review by dsousa12 — APPROVED** [review]
LGTM 🚀

**Pull request merged**